prac_07/programming_language.py

```diff
@@ -0,0 +1,18 @@
+class ProgrammingLanguage:
+    """Represent information about a programming language."""
+
```

Comment:
The ProgrammingLanguage class you've provided represents a programming language object with attributes such as name, typing, reflection, year, and pointer_arithmetic